lib/collections/schemas/shops.js

```diff
@@ -119,7 +119,7 @@ const Group = new SimpleSchema({
   slug: {
     type: String
   },
-  _id: {
+  groupId: {
```

Comment:
what's the thought behind this change? Using `groupId` instead of `_id` implicates to me that the groupId is pointing to another document or object elsewhere